### ISSUE TITLE
App crashes on iOS 18 when opening a recorded video

### ISSUE THREAD

**Issue by jeppster** [chadaustin/is-it-snappy#13]
After updating to iOS 18, if I attempt to open a previously recorded video the app crashes. 

Will also happen after it finishes recording and it tries open up that recording.

**Comment by chadaustin** [comment]
Thank you for the heads up. I don't have time to investigate right this moment but I'll try to get to it soon.

**Comment by yesbox** [comment]
Just reporting that iOS 18.1 did not fix it.

**Comment by chadaustin** [comment]
I have a new build in the App Store submission queue. In the meantime, if you would like to be added to TestFlight, let me know.

**Comment by jeppster** [comment]
I’d love to be added to TestFlight.

Adam

**Comment by chadaustin** [comment]
This is the first time I've used TestFlight's public invite link feature, so let me know if this link works:

https://testflight.apple.com/join/3snuCxtG

**Comment by jeppster** [comment]
Yes, it works! App installed just fine and app works as expected with no
crashing. Thank you so much!

**Comment by chadaustin** [comment]
Special thanks to @Exaphis for the fix.
